samples/csharp/MicrosoftEduGraphSamples/MicrosoftEduGraphSamples.csproj

```diff
@@ -12,6 +12,7 @@
     <PackageReference Include="Microsoft.Extensions.Configuration.Json" Version="6.0.0" />
     <PackageReference Include="Microsoft.Graph" Version="5.21.0" />
     <PackageReference Include="Microsoft.Graph.Beta" Version="5.46.0-preview" />
+    <PackageReference Include="Neo4jClient" Version="5.1.14" />
```

Comment:
where is this dependency coming from?

Reply:
That was added while trying to fix an issue. Removed as its no more needed.